topics/fair/tutorials/zenodo-in-galaxy/tutorial.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: tutorial_hands_on
+title: Uploading Data to Zenodo from Galaxy
+tags:
+  - fair
+  - zenodo
+questions:
+  - How to connect Zenodo to Galaxy?
+  - How to upload data from Galaxy to Zenodo
+objectives:
+  - Apply the integration of Zenodo in Galaxy
+  - Learn how to import records and files into Galaxy
+  - Apply the integration to export Galaxy Histories to Zenodo 
+  - Generate a DOI that can be cited by others.
+time_estimation: 30m
+key_points:
+  - Zenodo can be connected to Galaxy
+  - Records and files can be imported into Galaxy
+  - Galaxy Histories are directly exportable to Zenodo
+subtopic: fair-data
+contributions:
+  authorship:
+    - tflowers15 
+    - davelopez
+  funding:
+    - unimelb
+    - melbournebioinformatics
+    - AustralianBioCommons
+    - uni-freiburg
+    - eurosciencegateway
+    - mwk
+
+---
+
+# Overview
```

Comment:
```suggestion
```

please skip the heading, the subsequent "Discover..." will act as an abstract, otherwise it'll just show up as "overview" in previews which isn't as useful.